src/collection.ts

```diff
@@ -288,7 +288,7 @@ export class Collection<TSchema extends Document = Document> {
       options = {};
     }
 
-    // CSFLE passes in { w: 'majority' } to ensure the lib works in both 3.x and 4.x
+    // In use encryption passes in { w: 'majority' } to ensure the lib works in both 3.x and 4.x
```

Comment:
I think this should refer to the actual module name, instead of an encryption type for clarity